kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ProcessServiceBase.java

```diff
@@ -85,6 +85,22 @@ public String startProcess(String containerId, String processId, String marshall
         return response;
     }
 
+    public String startSynchronousProcess(String containerId, String processId, String payload, String marshallingType) {
+        containerId = context.getContainerId(containerId, ContainerLocatorProvider.get().getLocator());
+        // check validity of deployment and process id
+        definitionService.getProcessDefinition(containerId, processId);
+
+        logger.debug("About to unmarshal parameters from start spec parameters: '{}'", payload);
+        Map<String, Object> parameters = marshallerHelper.unmarshal(containerId, payload, marshallingType, Map.class);
+
+        logger.debug("Calling start sync process with id {} on container {} and parameters {}", processId, containerId, null);
+        Map<String, Object> outcome = processService.startSynchronousProcess(containerId, processId, parameters);
+
+        // return response
+        String response = marshallerHelper.marshal(containerId, marshallingType, outcome);
```

Comment:
It can be returned directly without creating an intermediate variable

Reply:
done